ChangeLog.zephyr.md

```diff
@@ -0,0 +1,12 @@
+# The List of changes to fit zephyr.
```

Comment:
Please squash the last two commits.

Reply:
@stephanosio 
Thank you for checking.
Rebasing done. Could you verify it?